docs/recce-cloud/share-recce-session-securely.md

```diff
@@ -17,25 +17,55 @@ Recce Cloud allows you to share your Recce validation results with full context,
 
 When you're ready to share your lineage exploration, query results, or validation checklist, simply click **Share** in Recce.
 
-The first time you do this, you'll need to connect your local Recce to Recce Cloud. This requires signing in and setting up an API token. Once connected, Recce Cloud will host your state file securely, allowing you to share a link that others can open in their browsers.
+The first time you do this, you'll need to connect your local Recce to Recce Cloud. This requires signing in and setting the connection of local Recce. Once connected, Recce Cloud will host your state file securely, allowing you to share a link that others can open in their browsers.
```

Comment:
> This requires signing in and setting the connection of local Recce.

Reading through the paragraph, and I feel the connection here is like a physical network connection.
Do you think "setting the association" is okay?

Reply:
Replace the wording `connect` with `associate` in this article.
Only keep the option `connect-to-cloud`, due to its related CLI implementation.